variants/frontend-base/template.rb

```diff
@@ -19,12 +20,18 @@
 
 package_json = JSON.parse(File.read("./package.json"))
 package_json["prettier"] = "prettier-config-ackama"
+package_json["browserslist"] = [
+  "defaults",
```

Comment:
I think we should document our default browser choice in the README we generate for the project eh? Sorry to add things on to your PR

Reply:
If its not already in this then I added our default requirements table to the readme template in the wiki https://github.com/ackama/wiki/wiki/Ackama-README-Template

Reply:
Currently this repo has a **copy** of the README template (mostly because builds here choke if prettier does not like the markdown). Maybe we should just `curl` it down instead? We would probably have to deal with build failures but maybe that's not a huge deal. Thoughts?

Reply:
@eoinkelly I think this would be good, but we should also set up a scheduled build in Travis to build once per month so that we catch these errors hopefully before a less experienced dev runs into it

Reply:
Actually @eoinkelly, this would not be super trivial since github.com/ackama/wiki/wiki is private and this is a public repository

Reply:
Perhaps we could move the README template somewhere public?

Reply:
Recorded the fetching of the README in https://github.com/ackama/rails-template/issues/41